bolt/onboarding_test.go

```diff
@@ -33,5 +33,6 @@ func initOnboardingService(f platformtesting.OnboardingFields, t *testing.T) (pl
 }
 
 func TestOnboardingService_Generate(t *testing.T) {
+	t.Skip("This service is not used, we use the kv bolt implementation")
```

Comment:
why don't you just delete this file?

Reply:
I'd prefer to get rid of these all together with something like https://github.com/influxdata/influxdb/pull/13039